brayns/Brayns.cpp

```diff
@@ -324,81 +324,55 @@ struct Brayns::Impl : public PluginAPI
-                        geometryParameters.getLoadCacheFile()});
-        pm.setProperty({"saveCacheFile", "saveCacheFile",
-                        geometryParameters.getSaveCacheFile()});
-        pm.setProperty({"circuitTargets", "circuitTargets",
-                        geometryParameters.getCircuitTargets()});
-        pm.setProperty({"circuitReport", "circuitReport",
-                        geometryParameters.getCircuitReport()});
-        pm.setProperty({"circuitMeshFolder", "circuitMeshFolder",
-                        geometryParameters.getCircuitMeshFolder()});
-        pm.setProperty({"circuitRandomSeed", "circuitRandomSeed",
-                        int32_t(geometryParameters.getCircuitRandomSeed())});
-        pm.setProperty({"circuitDensity", "circuitDensity",
-                        geometryParameters.getCircuitDensity()});
-        pm.setProperty({"circuitUseSimulationModel",
-                        "circuitUseSimulationModel",
-                        geometryParameters.getCircuitUseSimulationModel()});
-        pm.setProperty({"circuitMeshFilenamePattern",
-                        "circuitMeshFilenamePattern",
-                        geometryParameters.getCircuitMeshFilenamePattern()});
-        pm.setProperty({"radiusMultiplier", "radiusMultiplier",
-                        double(geometryParameters.getRadiusMultiplier())});
-        pm.setProperty({"radiusCorrection", "radiusCorrection",
-                        double(geometryParameters.getRadiusCorrection())});
-        pm.setProperty({"colorScheme", "colorScheme",
-                        enumToString(geometryParameters.getColorScheme()),
-                        enumNames<ColorScheme>()});
-        pm.setProperty({"geometryQuality", "geometryQuality",
-                        enumToString(geometryParameters.getGeometryQuality()),
-                        enumNames<GeometryQuality>()});
-        {
-            const auto layout = geometryParameters.getMorphologyLayout();
-
-            pm.setProperty({"morphologyLayoutNbColumns",
-                            "morphologyLayoutNbColumns",
-                            int32_t(layout.nbColumns),
-                            {0, std::numeric_limits<int32_t>::max()}});
-
-            pm.setProperty({"morphologyLayoutVerticalSpacing",
-                            "morphologyLayoutVerticalSpacing",
-                            int32_t(layout.verticalSpacing),
-                            {0, std::numeric_limits<int32_t>::max()}});
-
-            pm.setProperty({"morphologyLayoutHorizontalSpacing",
-                            "morphologyLayoutHorizontalSpacing",
-                            int32_t(layout.horizontalSpacing),
-                            {0, std::numeric_limits<int32_t>::max()}});
+            pm.addProperty({"boundingBoxP0", "boundingBoxP0", p0});
+            pm.addProperty({"boundingBoxP1", "boundingBoxP1", p1});
         }
-
+        pm.addProperty({"dampenBranchThicknessChangerate",
```

Comment:
`moreBeautifulBranches`? any name is probably terrible, do we need this option really?

Reply:
That's not what it means (which shows it's a bad name). I suggested lowPassSectionRadius, reduceRadiusVariation or similar, because that's what the option actually does. It's not reducing the change rate (samples stay where they are), but the difference between samples.